experiment/scheduler.py

```diff
@@ -19,6 +19,7 @@
 import sys
 import random
 import time
+import sqlite3
```

Comment:
nit: move this to line 19 for alpha order.

Reply:
fixed